src/domain/entities/Ref.ts

```diff
@@ -7,3 +9,11 @@ export interface Ref {
 export interface NamedRef extends Ref {
     name: string;
 }
+
+export const SharingSettingModel = Schema.object({
+    access: Schema.string,
+    id: Schema.dhis2Id,
+    name: Schema.string,
+});
+
+export type SharingSetting = GetSchemaType<typeof SharingSettingModel>;
```

Comment:
We decided to not use ``GetSchemaType`` instead we duplicate and enforce types on the model. Also the model should be in the data layer instead of domain